### PR TITLE
Counter Comment : Unit Testing   

### LINKED ISSUE #2: [3pt]  - Add tests for comments counter - Student B
Make sure that the function for the comments counter is covered with unit tests using Jest.

Make sure that you have tests for all edge cases.

### PR DESCRIPTION
**Pull request by AliShahSafdari**
Add Testing

- Add unit tests in the counter comment function
- Usage of ES6 Syntax and best practices.

### CODE REVIEW

File: src/test/counterComments.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { counterComment } = require('./counterComments.js');
+
+const items = [{ comment: 'Hi', creation_date: '2023-05-24', username: 'Ali' },
```
Comment (Mar12358) [inline src/test/counterComments.test.js]: - [ ] For this mock, don't you think it would be better if the name variable is more representative for this test? For example: `sixItems`, instead of `items`.[OPTIONAL]

File: src/test/counterComments.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  { comment: 'Hi', creation_date: '2023-05-24', username: 'Ali' },
+  { comment: 'so tasty', creation_date: '2023-05-24', username: 'Martin' },
+  { comment: 'Comments and more comments!', creation_date: '2023-05-24', username: 'Mr' },
+  { comment: 'Comments and more comments!', creation_date: '2023-05-24', username: 'Y' },
+  { comment: 'New comment', creation_date: '2023-05-24', username: 'Martin' }];
+
+const seconditems = [{ comment: 'Hi', creation_date: '2023-05-24', username: 'Ali' },
```
Comment (Mar12358) [inline src/test/counterComments.test.js]: - [ ] Please, use camelCase. [REQUIRED]

- [ ] You could also consider using a different name for the variable. In this case, for example, `twoItems` would be more, appropriate. It makes more sense grammatically. [OPTIONAL]

File: src/test/counterComments.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  { comment: 'Hi', creation_date: '2023-05-24', username: 'Ali' }];
+
+describe('Here we test the counter comments', () => {
+  test('Fist test: test the exact value ', () => {
```
Comment (Mar12358) [inline src/test/counterComments.test.js]: - [ ] Please, watch the Typo here for `Fist`, correct it to `First`

File: src/test/counterComments.test.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    expect(counterComment(items)).toBe(6);
+  });
+
+  test('Second test: test the not be equal', () => {
+    expect(counterComment(items)).not.toBe(3);
+  });
+
+  test('Thired test: test the not be equal', () => {
```
Comment (Mar12358) [inline src/test/counterComments.test.js]: - [ ] Please, watch the Typo here for `Thired`, correct it to `Third`.
- [ ] Please, watch the Description here for `not be equal`, correct it to `be equal` (This is the 'equal' test).

**Review by Mar12358 — CHANGES_REQUESTED** [review]
# Hi @AliShahSafdari ,
### Good job so far! You are almost there!
# Required Changes ♻️
- Minor changes required. Please, take a look at the comments below.

**Happy Coding 🎉**

**Review by Mar12358 — APPROVED** [review]
Congratulations! APPROVED!
Time to merge! 💯
